File: app/src/main/java/droiddevelopers254/droidconke/viewmodels/StarrViewModel.java
package droiddevelopers254.droidconke.viewmodels;

import androidx.lifecycle.ViewModel;

import droiddevelopers254.droidconke.repository.RoomStarrSessionRepo;

public class StarrViewModel extends ViewModel {
    private RoomStarrSessionRepo roomStarrSessionRepo;

    public StarrViewModel(){
        roomStarrSessionRepo = new RoomStarrSessionRepo();
    }

}

File: app/src/main/java/droiddevelopers254/droidconke/viewmodels/StarrViewModel.kt
package droiddevelopers254.droidconke.viewmodels

import androidx.lifecycle.ViewModel
import droiddevelopers254.droidconke.repository.RoomStarrSessionRepo

class StarrViewModel : ViewModel() {
    private val roomStarrSessionRepo: RoomStarrSessionRepo = RoomStarrSessionRepo()

}
